packages/amplify-cli/src/plugin-helpers/access-plugins-file.ts

```diff
@@ -63,8 +63,8 @@ function getPluginsJsonFileName(): string {
   let result = constants.PLUGINS_FILE_NAME;
   const amplifyExecutableName = path.basename(process.argv[1]);
 
-  if (amplifyExecutableName !== constants.Amplify) {
-    result = amplifyExecutableName + '-' + constants.PLUGINS_FILE_NAME;
+  if (amplifyExecutableName === 'amplify-dev') {
```

Comment:
do we need to change this back? or is this intentional?

Reply:
This is done intentionally. 
We don't want to produce too many of those plugins.json files.

Reply:
So shouldn't it be `amplify-dev || amplify`